README.md

```diff
@@ -1 +1,3 @@
-# sclorg-testing-farm
\ No newline at end of file
+# Testing Farm FMF testing plans
```

Comment:
I would prefer to mention here something like:

`FMF testing plans for using it in Testing Farm`.